torchvision/utils.py

```diff
@@ -125,12 +125,14 @@ def save_image(
         tensor (Tensor or list): Image to be saved. If given a mini-batch tensor,
             saves the tensor as a grid of images by calling ``make_grid``.
         fp (string or file object): A filename or a file object
+        normalize (bool, optional): If True, shift the image to the range (0, 1),
+            by the min and max values specified by :attr:`range`. Default: ``False``.
```

Comment:
Why would we expose `normalize` here but not any of the other parameters of `make_grid`? Would not it be better to remove the parameter and let it be captured by `kwargs `? This way your change will be backwards-compatible, we will improve consistency by allowing all the args of make_grid to be captured by `kwargs` and we won't have to document the same variable twice. Thoughts?